app-editors/bvi/files/bvi-1.4.1-prototypes.patch

```diff
@@ -0,0 +1,32 @@
+diff --git a/bmore.h b/bmore.h
```

Comment:
Can you include the link to the upstream report here please?

Reply:
Done.